Extension/src/LanguageServer/client.ts

```diff
@@ -344,7 +345,6 @@ class DefaultClient implements Client {
             languageClient.registerProposedFeatures();
             languageClient.start();  // This returns Disposable, but doesn't need to be tracked because we call .stop() explicitly in our dispose()
             util.setProgress(util.getProgressExecutableStarted());
-            this.rootFolder = workspaceFolder;
```

Comment:
remove this order of initialization dependency by initializing this first, and then don't pass the local variable in to `createLanguageClient`.

Reply:
Uh...why does that matter?

Reply:
because the intuitive place to initialize member variables is in the constructor.

Reply:
Ok